src/commands/aksAccount/aksAccount.ts

```diff
@@ -89,10 +82,13 @@ export async function selectSubscriptions(): Promise<void> {
             },
         };
     });
+    
+    // show picked items at the top
+    quickPickItems.sort((a, b) => (a.picked === b.picked ? 0 : a.picked ? -1 : 1));
```

Comment:
💡 Please note: `(a, b)` are objects lets name them properly, aka what `a` and `b` signifies. :)

Reply:
This is now updated